app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PostsHelper
+end
```

Comment:
- [ ] You were requested not to use file generation command like scaffold command, but you used it anyways. Kindly remove this scaffold file.